doc/source/data/doc_code/quick_start.py

```diff
@@ -1,64 +1,119 @@
 # flake8: noqa
 # fmt: off
 
-# __data_setup_begin__
-
+# __create_from_python_begin__
 import ray
 
-# Create a Dataset of Python objects.
+# Create a Dataset from Python objects, which are also held as Python objects.
 ds = ray.data.range(10000)
 # -> Dataset(num_blocks=200, num_rows=10000, schema=<class 'int'>)
 
 ds.take(5)
 # -> [0, 1, 2, 3, 4]
 
-ds.count()
-# -> 10000
+ds.schema()
+# <class 'int'>
 
-# Create a Dataset of Arrow records.
-ds = ray.data.from_items([{"col1": i, "col2": str(i)} for i in range(10000)])
-# -> Dataset(num_blocks=200, num_rows=10000, schema={col1: int64, col2: string})
+# Create a Dataset from Python objects, which are held as Arrow records.
+ds = ray.data.from_items([
+        {"sepal.length":5.1,"sepal.width":3.5,"petal.length":1.4,"petal.width":0.2,"variety":"Setosa"},
+        {"sepal.length":4.9,"sepal.width":3.0,"petal.length":1.4,"petal.width":0.2,"variety":"Setosa"},
+        {"sepal.length":4.7,"sepal.width":3.2,"petal.length":1.3,"petal.width":0.2,"variety":"Setosa"},
+     ])
+# -> Dataset(num_blocks=3, num_rows=3, schema={sepal.length: double, sepal.width: double, petal.length: double, petal.width: double, variety: string})
```

Comment:
Some of these lines will be very long, we should wrap these at 88 characters, or maybe at 80, since it looks like 88 doesn't cut it because our code block width is set to horizontal scroll at 88 characters: https://github.com/ray-project/ray/pull/24831#discussion_r874312681

Reply:
Yep. Please cut these manually. It looks like since there are no spaces, the word wrap css style doesn't apply.

So an easy fix is to format the json properly (add spaces after the commas).

Reply:
Reformatted.